src/components/KCopy/KCopy.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div class="k-copy">
+    <span
+      v-if="badge"
+      class="copy-badge-text"
+    >
+      {{ badgeText }}
+    </span>
+    <div
+      :class="[
+        'copy-container',
+        truncateStyles,
+        badgeStyles
+      ]"
+    >
+      <KTooltip
+        v-if="format !== 'hidden'"
+        class="copy-text"
+        :class="[truncateContent, useMono]"
+        data-testid="copy-text"
+        :label="textTooltip"
+        placement="bottomStart"
+        position-fixed
+      >
+        <span>
+          {{ textFormat }}
+        </span>
+      </KTooltip>
+
+      <KTooltip
+        class="text-icon-wrapper"
+        :label="`${props.badgeText}` ? iconTitle : tooltipText"
+        max-width="500px"
```

Comment:
question: should the max-width be configurable?

Reply:
Current implementation of CopyUuid had it set to 500px, I will add in a prop to make it configurable.